index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sameed - Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+</head>
+
+
+
+<body>
+
+    <header>
+        <nav>
+            <div class="logo-container">
+                <img src="images for portfolio/logo.jpg" alt="" class="logo">
+                <a class="logo-text">Sameed Mohsin</a>
+            </div>
+
+            <div class="menu">
+                <img src="./Images for portfolio/Hamburger.svg" alt="" class="hamburger" width="50px" height="50p">
+            </div>
+        </nav>
+    </header>
+
+    <section>
+        <div>
+            <h1 class = "top-heading"> I'm Sameed</h1>
+        </div>
+        <div class="my-description">
+            <p>I am a web developer.</p>
+            <p>You can count on me for building, developing, and maintaining your websites.</p>
+        </div>
+
+    </div>
+    
+</section>
+
+<footer>
+    <div class="connect">
+        <ul>
+            Connect to me
+            <li>
+                <a href="#" class="icon">
+                    <img class="github" src="./Images for portfolio/github.svg" alt="GitHub">
+                </a>
+            </li>
+            <li>
+                <a href="#" class="icon">
+                    <img src="./Images for portfolio/Twitter.svg" alt="Twitter" class="twitter">
+                </a>
+            </li>
+            <li>
+                <a href="#" class="icon">
+                    <img src="./Images for portfolio/Linkedin.svg" alt="LinkedIn" class="linkedin">
+                </a>
+            </li>
+        </ul>
```

Comment:
You will align this `ul`  with flexbox. To read about flexbox [visit](https://yoksel.github.io/flex-cheatsheet/#section-align-items-self)
This is a very comprehensive guide. Learn from it. Watch some youtube tutorials, and then apply flexbox on it, to align these social-media icons horizontally.